.gitignore

```diff
@@ -1 +1,2 @@
 /build
+/tools/utils/__pycache__
```

Comment:
My bad, thanks for the cleanup!

Might be better to specify just `__pycache__` (without the path) so it applies to all directories.

Reply:
Good point